packages/components/src/components/scrollable/scrollable.stories.ts

```diff
@@ -43,148 +47,146 @@ export default {
 };
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
-export const ButtonWithGradient = {
-  parameters: { controls: { exclude: ['buttons'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      args,
-      constants: [{ type: 'attribute', name: 'buttons', value: true }]
-    });
-  }
-};
+/**
+ * Use the `orientation` attribute to set the scroll direction of the component.
+ */
 
-export const Shadow = {
-  parameters: { controls: { exclude: ['shadows'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      args,
-      constants: [{ type: 'attribute', name: 'shadows', value: 'true' }]
-    });
-  }
+export const Orientation = {
+  name: 'Orientation',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-scrollable orientation="horizontal">
+        <div class="flex h-40 items-start w-max p-4 gap-1">
+          <sd-tag size="lg">Tag 1</sd-tag>
+          <sd-tag size="lg">Tag 2</sd-tag>
+          <sd-tag size="lg">Tag 3</sd-tag>
+          <sd-tag size="lg">Tag 4</sd-tag>
+          <sd-tag size="lg">Tag 5</sd-tag>
+          <sd-tag size="lg">Tag 6</sd-tag>
+        </div>
+      </sd-scrollable>
+
+      <sd-scrollable orientation="vertical">
```

Comment:
Could you reduce the height so that e.g. the 6th line is cut-off in the middle?

![image](https://github.com/user-attachments/assets/d891dc59-7682-4e6a-bd7f-504ea9ab08eb)